### ISSUE TITLE
empty map (no records) showing on default home page, in mapping app

### ISSUE THREAD

**Issue by jmckenna** [iodepo/oih-ui#145]
- I am sure this is by design, however I find it very odd that on the default view, the lower mapping app just shows the globe, and you cannot see any record geometry/points/polygons etc.
- you can however interact with that map, zoom-in etc, and search for those missing points and polygons, which really confuses the user
- what is the point of having a full (and empty) mapping app on the default view? 
- I suggest to fully remove the mapping app on the default view, or, just have a small screen capture of a map showing some of the geometries, that when the user clicks it, it automatically opens the full map viewer

source: https://oih.ioc-africa.org/

![Screenshot 2024-09-13 at 2 44 54 PM](https://github.com/user-attachments/assets/6176f003-700a-44b5-b719-7eb2f6d0fc1a)

Thoughts?

**Comment by pbuttigieg** [comment]
Yes, there should be some content on the base layer, like the heatmap or hex cells 

I wouldn't remove it.

**Comment by emarzini** [comment]
Hello, it seems that the Africa's portal frontend is not updated at the last release as the main one

**Comment by arnounesco** [comment]
@emarzini we do not touch the African portal installation for the time being, we are installing everything on our dev server and are trying to reinstall the complete suite (not only the UI, but also indexing and harvesting) on that server before updating the Africa installation.
our test installation is either on dev.oih.iode.org or devsearch.oceaninfohub.org
